tests/SprykerEcoTest/Zed/Unzer/_support/UnzerZedTester.php

```diff
@@ -0,0 +1,565 @@
+        $this->setConfig(UnzerConstants::UNZER_CHARGE_RETURN_URL, 'https://spryker.com');
+        $this->setConfig(UnzerConstants::WEBHOOK_RETRIEVE_URL, 'https://spryker.com');
+        $this->setConfig(UnzerConstants::MASTER_MERCHANT_PARTICIPANT_ID, '111111');
+        $this->setConfig(UnzerConstants::MAIN_REGULAR_KEYPAIR_ID, 'id');
+        $this->setConfig(UnzerConstants::VAULT_DATA_TYPE, 'unzer-private-key');
+        $this->setConfig(VaultConstants::ENCRYPTION_KEY, 'key');
+    }
+
+    /**
+     * @return \SprykerEco\Zed\Unzer\UnzerConfig
+     */
+    public function createConfig(): UnzerConfig
+    {
+        return new UnzerConfig();
+    }
+
+    /**
+     * @return \SprykerEco\Zed\Unzer\Persistence\UnzerEntityManager
+     */
+    public function createEntityManager(): UnzerEntityManager
+    {
+        return new UnzerEntityManager();
+    }
+
+    /**
+     * @return \SprykerEco\Zed\Unzer\Persistence\UnzerRepository
+     */
+    public function createRepository(): UnzerRepository
+    {
+        return new UnzerRepository();
+    }
+
+    /**
+     * @return \Generated\Shared\Transfer\QuoteTransfer
+     */
+    public function createQuoteTransfer(): QuoteTransfer
+    {
+        $quoteTransfer = (new QuoteBuilder())
+            ->withItem()
+            ->withTotals()
+            ->withShippingAddress()
+            ->withBillingAddress()
+            ->withStore()
+            ->build();
+
+        return $quoteTransfer
+            ->setCustomer($this->haveCustomer())
+            ->setStore($this->haveStore())
+            ->setPayment($this->createPaymentTransfer());
+    }
+
+    /**
+     * @return QuoteTransfer
+     */
+    public function createMarketplaceQuoteTransfer(): QuoteTransfer
+    {
+        $quoteTransfer = (new QuoteBuilder())
+            ->withItem([
+                'merchantReference' => static::MERCHANT_REFERENCE
+            ])
+            ->withItem([
+                'merchantReference' => null
+            ])
+            ->withTotals()
+            ->withShippingAddress()
+            ->withBillingAddress()
+            ->build();
+
+        return $quoteTransfer
+            ->setCustomer($this->haveCustomer())
+            ->setPayment($this->createPaymentTransfer())
+            ->setStore($this->haveStore());
+    }
+
+    /**
+     * @return \Generated\Shared\Transfer\PaymentTransfer
+     */
+    public function createPaymentTransfer(): PaymentTransfer
+    {
+        return (new PaymentTransfer())
+            ->setPaymentProvider(static::PAYMENT_PROVIDER)
+            ->setPaymentMethod(static::PAYMENT_METHOD)
+            ->setPaymentSelection(static::PAYMENT_METHOD)
+            //@todo dynamically change types for different payment methods
+            ->setUnzerPayment($this->createUnzerPaymentTransfer(true, true));
+    }
+
+    /**
+     * @param bool $isMarketplace
+     * @param bool $isAuthorizable
+     *
+     * @return UnzerPaymentTransfer
```

Comment:
```suggestion
     * @return Generated\Shared\Transfer\UnzerPaymentTransfer
```

Please check similar cases below.